src/main/java/de/tum/in/www1/artemis/service/connectors/localci/LocalCIBuildJobExecutionService.java

```diff
@@ -345,7 +297,7 @@ private boolean isValidTestResultFile(TarArchiveEntry tarArchiveEntry) {
         String result = (lastIndexOfSlash != -1 && lastIndexOfSlash + 1 < name.length()) ? name.substring(lastIndexOfSlash + 1) : name;
 
         // Java test result files are named "TEST-*.xml", Python test result files are named "*results.xml".
-        return !tarArchiveEntry.isDirectory() && ((result.endsWith(".xml") && result.startsWith("TEST-")) || result.endsWith("results.xml"));
+        return !tarArchiveEntry.isDirectory() && result.endsWith(".xml");
```

Comment:
```suggestion
        return !tarArchiveEntry.isDirectory() && result.endsWith(".xml") && !result.endsWith("pom.xml");
```

Reply:
Java Maven exercises won't work otherwise as the pom.xml will be considered a test result file